### PR TITLE
Feat(Load Balancer):  create documentation for using an external certificate with load balancer

### PR DESCRIPTION
**Pull request by OvhValentin**
Refer to SK-1801.

PR to create a guide for using an SSL certificate generated by a third party.

### CODE REVIEW

**Comment by OvhValentin** [comment]
I need to edit the index file

File: pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+- A [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account
+- [Preparing your environment for using the OpenStack API](/pages/public_cloud/compute/prepare_the_environment_for_using_the_openstack_api)
+- [OpenStack Octavia client](https://docs.openstack.org/python-octaviaclient/latest/install/index.html) and [OpenStack Barbican](https://docs.openstack.org/python-barbicanclient/latest/install/index.html) set up
+- A certificate from a third-party supplier.
+
+If you are not yet familiar with creating a Load Balancer, please follow our guide on [Getting started with Load Balancer on Public Cloud](/pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service) before you continue with this tutorial.
+
+## Instructions
+
+### Attaching a Floating IP address to a Load Balancer
```
Comment (yomovh) [inline pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md:40]: This is already describe in /pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service why add it here ?

File: pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+You need to create a PKCS#12 package with your certificate inside:
+
+```bash
+sudo openssl pkcs12 -export -inkey <private_key.key> -in <main_certificate.pem> -out domain.tld.p12
+```
+
+You must download this file directly onto your computer in order to be able to send it to Openstack Barbican (‘Secret as a Service’).
+
+```bash
```
Comment (yomovh) [inline pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md:71]: Add how to do the same through Horizon

File: pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+### Configuring your Load Balancer
+
+
+/// details | Via Horizon
+
+In the Horizon interface, open the `Network`{.action} section, then click `Load Balancers.`{.action}
+
+Select your Load Balancer by clicking on `the name of your Load Balancer.`{.action}
+
+You can now click on the `Listerners`{.action} section, then on `Create Listener` as follows:
```
Comment (yomovh) [inline pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md]:
```suggestion
You can now click on the `Listeners`{.action} section, then on `Create Listener` as follows: 
```

File: pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+![l7policy creation](images/listenerpolicieslist.png){.thumbnail}
+
+In this window, you need to name the L7 Policy, choose the value `REDIRECT_TO_URL`{.action} in the ‘Action’ field and then `write its domain name`{.action} in the ‘Redirect URL’ field. You can click on `Create L7 policy`{.action} button at the bottom of the window to create your L7 Policy.
+
+![l7policy creation](images/l7policycreation.png){.thumbnail}
+
+///
+
+/// details | Via the OpenStack API
```
Comment (yomovh) [inline pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md]: Here you are missing the manager interface ;-)
What I would do is to link this page /pages/public_cloud/public_cloud_network_services/getting-started-01-create-lb-service) and says  create a REDIRECT_TO_URL policy with the interface of your choice !

File: pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+![listeners https pool details](images/listenerhttpscreation2.png){.thumbnail}
+
+In the third section, called ‘Pool Members’, select the corresponding instance and press the `Add`{.action} button at the end of the line. Once this has been done, select port `80.`{.action} as the ‘Port’.
+
+![listeners https pool members details](images/listenerhttpscreation3.png){.thumbnail}
+
+For the ‘Monitor Details’ section, we're going to click `No`{.action} as we won't be demonstrating this part in this guide.
+
+Now go to the section called `SSL Certificates`{.action} and select the certificate that was previously added to Openstack Barbican, [in this part](#preparingcertificate).
+You can click `Create L7 policy`{.action} button at the bottom of the window to create your configured Load Balancer.
```
Comment (yomovh) [inline pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md]: `Create L7 policy`  does not sound right

**Review by yomovh — COMMENTED** [review]
Here are a few comments

File: pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md (hunk 2 above)
Comment (OvhValentin) [reply pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md:71]: to my knowledge, sending a secret to Barbican is only possible via openstack api. no?

Comment (yomovh) [reply pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/guide.en-gb.md:71]: you are right only CLI is possible.

**Comment by OvhValentin** [comment]
@yomovh Do you validate the doc ?

File: pages/index.md [hunk 6]
```diff
@@ -682,6 +682,7 @@
             + [Updating a Load Balancer size via the Horizon interface](public_cloud/public_cloud_network_services/update-load-balancer-size)
         + [Tutorials](public-cloud-network-tutorials)
             + [Configuring a secure Load Balancer with Let's Encrypt](public_cloud/public_cloud_network_services/tutorials-01-secure-lb-letsencrypt)
+            + [Configuring a `TERMINATED_HTTPS` listener via CLI / Horizon](pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat)
```
Comment (Y0Coss) [inline pages/index.md]: I do not recommend adding quotes in the index and in a guide title.

File: pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,2 @@
+id: 971a9f5e-3149-4fba-9b70-c94629dcdbcf
+full_slug: public-cloud-network-load-balancer-third-party-certificat
```
Comment (Y0Coss) [inline pages/public_cloud/public_cloud_network_services/tutorials-02-secure-lb-external-certificat/meta.yaml:2]:
```suggestion
full_slug: public-cloud-network-load-balancer-third-party-certificate
```

**Comment by yomovh** [comment]
> @yomovh Do you validate the doc ?

LTGM

**Pull request merged**